internal/test/util/kong_config.go

```diff
@@ -69,6 +78,10 @@ func fetchKongConfig() (KongConfig, error) {
 	if err != nil {
 		return KongConfig{}, fmt.Errorf("fetch SNIs: %v", err)
 	}
+	vaults, err := client.Vaults.ListAll(ctx)
+	if err != nil {
+		return KongConfig{}, fmt.Errorf("fetch Vaults: %v", err)
```

Comment:
Only newer DPs have the `/vaults` endpoint, so returning an error here would make all tests to fail ([example](https://github.com/Kong/koko/actions/runs/3340568631/jobs/5530817665))

Reply:
I was thinking that since you had added `kongClient.RunWhenKong(t, ">= 3.0.0")` to the vault tests that this wouldn't be a problem, but I forgot about all of the other tests that use this.